circuits/circom/processMessages.circom

```diff
@@ -124,6 +127,7 @@ template ProcessMessages(
     var msgTreeZeroValue = 8370432830353022751713833565135785980866757267633941821328460903436894336785;
 
     // Verify currentSbCommitment
+    // currentSbCommitment === hash3(currentStateRoot, currentBallotRoot, currentSbSalt)
```

Comment:
Not sure I understand this comment - is the pseudo code for what's happening?

Reply:
correct! basically the current state and ballot trees commitment will be the hash of (currentStateRoot, currentBallotRoot, currentSbSalt)